DQM/DTMonitorModule/src/DTChamberEfficiency.cc

```diff
@@ -79,7 +79,7 @@ DTChamberEfficiency::DTChamberEfficiency(const ParameterSet& pSet)
   edm::ConsumesCollector iC = consumesCollector();
 
   theMeasurementExtractor = new MuonDetLayerMeasurements(thedt4DSegments,thecscSegments,
-  							 labelRPCRecHits,labelGEMRecHits,labelME0RecHits,iC,true,false,false,false,false); 
+  							 labelRPCRecHits,labelGEMRecHits,labelGEMRecHits,labelME0RecHits,iC,true,false,false,false,false); 
```

Comment:
It seems like `labelGEMRecHits` doesn't need to be a member of this class since it is never filled or used in a nontrivial way. It could just be replaced here with a default `edm::InputTag()`, as is done elsewhere in this PR.